core/Metadatacore.rdf

```diff
@@ -32,8 +32,8 @@
 		<owl:imports rdf:resource="https://spec.industrialontologies.org/ontology/core/MappingCommonsToIOFCore/"/>
 		<owl:imports rdf:resource="https://spec.industrialontologies.org/ontology/core/commonstocoremapping/meta/MappingAnnotationVocabularyToCommons/"/>
 		<owl:imports rdf:resource="https://spec.industrialontologies.org/ontology/core/meta/AnnotationVocabulary/"/>
-		<owl:versionIRI rdf:resource="https://spec.industrialontologies.org/ontology/20230120/core/Metadatacore/"/>
-		<iof-av:copyright>Copyright (c) 2023 Open Applications Group</iof-av:copyright>
+		<owl:versionIRI rdf:resource="https://spec.industrialontologies.org/ontology/master/202301/core/Metadatacore/"/>
```

Comment:
Why is there the word 'master' in the IRI?

Reply:
> Why is there the word 'master' in the IRI?

See https://github.com/iofoundry/ontology/pull/40#issuecomment-1412038889 above and @wsobel reply.